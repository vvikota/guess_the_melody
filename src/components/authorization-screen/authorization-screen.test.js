import React from 'react';
import renderer from 'react-test-renderer';
import AuthorizationScreen from './authorization-screen';

it(`AuthorizationScreen renders`, ()=> {
  const tree = renderer
    .create(<AuthorizationScreen
      onChange={jest.fn()}
      logIn={jest.fn()}
      email={`test@test.ru`}
      password={`pass`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
