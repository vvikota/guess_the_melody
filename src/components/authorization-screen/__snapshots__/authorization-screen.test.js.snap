// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorizationScreen renders 1`] = `
<section
  className="login"
>
  <div
    className="authorization-pic"
  >
    <img
      alt="logo"
      src="./img/authorization.png"
      width="140"
    />
  </div>
  <h2
    className="login__title"
  >
    Необходима авторизация
  </h2>
  <p
    className="login__text"
  >
    Представтесь:
  </p>
  <form
    action=""
    className="login__form"
  >
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="email"
      >
        Email
      </label>
      <input
        className="login__input"
        id="email"
        name="email"
        onChange={[MockFunction]}
        type="text"
      />
    </p>
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="password"
      >
        Пароль
      </label>
      <input
        className="login__input"
        id="password"
        name="password"
        onChange={[MockFunction]}
        type="text"
      />
      <span
        className="login__error"
      >
        Неверный пароль
      </span>
    </p>
    <button
      className="login__button button"
      onClick={[MockFunction]}
      type="submit"
    >
      Войти
    </button>
  </form>
</section>
`;
